onnxruntime/test/contrib_ops/quantize_attention_op_test.cc

```diff
@@ -217,8 +254,43 @@ static void RunQAttentionAll(
   RunQAttentionCUDA(input_data, weight_data, bias_data, mask_index_data, output_data,
                     batch_size, sequence_length, hidden_size, number_of_heads,
                     use_special_quantize_parameter, is_unidirectional, use_float16, input_hidden_size);
+  RunQAttentionDNNL(input_data, weight_data, bias_data, mask_index_data, output_data,
```

Comment:
this can only be called if DNNL is enabled.
RunQAttentionAll() is called from various unit tests.
i think that's what's causing the test failures for the non-DNNL CI's.

Reply:
That was exactly what the problem was. Thanks for taking a look. I have pushed a fix.